api/app/Policies/PoolPolicy.php

```diff
@@ -211,7 +211,9 @@ public function archiveAndUnarchive(User $user, Pool $pool)
     {
         $pool->loadMissing('team');
 
-        return $user->isAbleTo('archive-team-pool', $pool->team);
+        // TODO: should this check for non-draft status?
+
+        return $user->isAbleTo('archive-team-publishedPool', $pool->team);
```

Comment:
I am not sure if you'd ever want to archive a draft pool, but I don't see a reason to explicitly block it

Reply:
I think its more straightforward if we restrict deleting to draft pools and archiving to published pools.